### PR TITLE
glib2: Update to 2.80.0

### PR DESCRIPTION
**Pull request by lazka**
Remove patch included upstream.
Add new python-packaging dep for gdbus-codegen
Remove gtk-doc dep, no longer used and we don't build docs anyway atm

Disable introspection for bootstrapping

### CODE REVIEW

**Comment by Biswa96** [comment]
Would it be possible to add a `_enable_gir=no` flag (or something similar) for the bootstrapping? The similar approach was done with mingw gcc.

**Comment by lazka** [comment]
sure, I'll try, but in a future PR.

**Comment by Biswa96** [comment]
waiting...

File: mingw-w64-glib2/PKGBUILD [hunk 1]
```diff
@@ -17,31 +17,30 @@ depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
          "${MINGW_PACKAGE_PREFIX}-gettext-runtime"
          "${MINGW_PACKAGE_PREFIX}-pcre2"
          "${MINGW_PACKAGE_PREFIX}-libffi"
+         "${MINGW_PACKAGE_PREFIX}-python-packaging"
```
Comment (Biswa96) [inline mingw-w64-glib2/PKGBUILD:20]: Should python-packaging be in optdepends like ArchLinux does?

Comment (lazka) [reply mingw-w64-glib2/PKGBUILD:20]: I saw that, but it wasn't clear to me why it should be.

**Pull request merged**